### PR TITLE
New d2d collab pages

### PR DESCRIPTION
**Pull request by andywong418**
## :pushpin: References

- **Wonder issue:**
- **Related pull-requests:** https://github.com/wondrous-dev/wondrous-backend/pull/657

## :movie_camera: Screenshot or Video
https://www.loom.com/share/bc6dd84915174157b69b882c8cccd002
## :cake: Checklist:

- [ ] I self-reviewed my changes
- [ ] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [ ] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-app/components/Collaboration/SharedOrgHeader/index.tsx [hunk 1]
```diff
@@ -4,7 +4,85 @@ import { SafeImage } from 'components/Common/Image';
...
+export const SharedOrgHeaderCard = (props) => {
+  const { collab, blurChild, blurParent } = props;
+  return (
+    <Grid
+      container
+      direction="row"
+      wrap="nowrap"
+      gap="8px"
+      sx={{ width: 'auto' }}
+      style={{
```
Comment (Lamperoyge) [inline wondrous-app/components/Collaboration/SharedOrgHeader/index.tsx:18]: We can pass width and alignItems directly to Grid as props, something like `<Grid alignItems="center" width="auti" .../>`

File: wondrous-app/components/Collaboration/SharedOrgHeader/styles.tsx [hunk 2]
```diff
@@ -16,3 +16,41 @@ export const Separator = styled(Typography)`
...
+export const SeparatorDiv = styled.div`
+  width: 16px;
+  height: 16px;
+  border-radius: 8px;
+`;
+
+export const OrgName = styled(Typography)`
+  && {
+    font-weight: 500;
+    font-family: Space Grotesk;
```
Comment (Lamperoyge) [inline wondrous-app/components/Collaboration/SharedOrgHeader/styles.tsx:44]: `font-family: ${typography.fontFamily}`

File: wondrous-app/components/Common/button.tsx [hunk 3]
```diff
@@ -20,6 +20,10 @@ const ButtonInner = styled.button`
   color: white;
   padding: 12px;
   cursor: pointer;
+  font-family: Space Grotesk;
```
Comment (Lamperoyge) [inline wondrous-app/components/Common/button.tsx:23]: same as above

File: wondrous-app/components/organization/collaborations/CollaborationItem.tsx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+const ORG_PERMISSIONS = {
+  MANAGE_SETTINGS: 'manageSettings',
+  CONTRIBUTOR: 'contributor',
+};
+
+export const ActiveCollaborationItem = (props) => {
+  const { collab, userPermissionsContext } = props;
+  const [orgRoleName, setOrgRoleName] = useState(null);
+  const [permissions, setPermissions] = useState(undefined);
+  useEffect(() => {
```
Comment (Lamperoyge) [inline wondrous-app/components/organization/collaborations/CollaborationItem.tsx:34]: I wonder if we can avoid state & effect here and just have memoized values. So something like 

` const orgPermissions = parseUserPermissionContext({
      userPermissionsContext,
      orgId: collab?.id,
    });


  const permissions = useMemo(() => {
    if (
      orgPermissions?.includes(PERMISSIONS.MANAGE_MEMBER) ||
      orgPermissions?.includes(PERMISSIONS.FULL_ACCESS) ||
      orgPermissions?.includes(PERMISSIONS.APPROVE_PAYMENT)
    ) {
      return ORG_PERMISSIONS.MANAGE_SETTINGS;
    } else if (
      userPermissionsContext?.orgPermissions &&
      collab?.id in userPermissionsContext?.orgPermissions &&
      orgPermissions
    ) {
      // Normal contributor with no access to admin settings
      return ORG_PERMISSIONS.CONTRIBUTOR
    }
  }, [orgPermissions, userPermissionsContext, collab?.id]);
  const orgRoleName = useMemo(() => userPermissionsContext?.orgRoles?[collab?.id], [collab?.id, userPermissionsContext]);`

File: wondrous-app/components/organization/collaborations/styles.tsx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+export const CollabCardHeader = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const CollabDescription = styled(Typography)`
+  && {
+    color: ${palette.grey250};
+    font-size: 15px;
+    font-family: Space Grotesk;
```
Comment (Lamperoyge) [inline wondrous-app/components/organization/collaborations/styles.tsx:93]: font-family: ${typography.fontFamily}

File: wondrous-app/components/Collaboration/SharedOrgHeader/index.tsx (hunk 1 above)
Comment (andywong418) [reply wondrous-app/components/Collaboration/SharedOrgHeader/index.tsx:18]: https://mui.com/material-ui/api/grid/ doesn't seem like it?

**Pull request merged**